src/YHttpWidgetsActionHandler.h

```diff
@@ -158,6 +163,26 @@ class YHttpWidgetsActionHandler : public YHttpHandler
             }
         } );
     }
+    
+    template<typename T>
+    int get_menu_selector_handler( T *widget, const std::string &value, std::ostream& body) {
+        return action_handler<T>( widget, body, [&] (T *menu_selector) {
+            // Vector of string to store path to the tree item
+            std::vector<std::string> path;
+            boost::split( path, value, boost::is_any_of( TreePathDelimiter ) );
+            YMenuItem * item = menu_selector->findItem( path );
+            if ( item )
+            {
+                yuiMilestone() << "Activating Item by path :" << value << " in \"" << menu_selector->label() << "\" menu selector" << std::endl;
+                menu_selector->setKeyboardFocus();
+                activate_widget( menu_selector, item );
```

Comment:
Do we need extra event here to activate it? Because I doubt that it's implemented for `YMenuBar`

Reply:
When I tested I could see that it can click on disabled and enabled items, like YMenuButton probably does.

Reply:
Ah, right, as it's inherits from it.